spring-cloud-generator/generate-library-list.sh

```diff
@@ -1,13 +1,7 @@
 #!/bin/bash
 
-#cmd line:: ./generate-library-list.sh -c v1.1.0
-
-while getopts c: flag
-do
-    case "${flag}" in
-        c) commitish=${OPTARG};;
-    esac
-done
+#cmd line:: ./generate-library-list.sh
```

Comment:
Let's also remove this change since it's already made in #1499?

Reply:
Change reverted - thanks!